ledger/index.ts

```diff
@@ -151,7 +151,7 @@ export async function signLedgerTaprootBtcTransaction(
   transport: Transport,
   network: NetworkType,
   addressIndex: number,
-  recipient: Recipient,
+  recipient: Array<Recipient>,
```

Comment:
Let's change the `recipient` to `recipients` where it's an array and this PR can be merged